src/server/main.py

```diff
@@ -18,11 +18,12 @@ def read_root():
 def generate_prompt(country: str):
     user_query = f"What is the weather in {country}?"
     output = llm_generate_prompt(MODEL, user_query)
-    return {"Message": output}
+    return {"Message": output["response"]}
 
 
 def llm_generate_prompt(model, query):
     print("Start generate response from LLM", flush=True)
     response = client.generate(model=model, prompt=query)
+    response.pop("context")
```

Comment:
[![Once You Pop (You Can't Stop) ](http://img.youtube.com/vi/rcqQcoKI_xQ/0.jpg)](http://www.youtube.com/watch?v=rcqQcoKI_xQ "Once You Pop (You Can't Stop) ")

Reply:
I don't want to see that junk, get it out of my screen!
<div><img src="https://media0.giphy.com/media/C5ZIna5oroan9cdHz9/giphy.gif?cid=5a38a5a2ndfpe5n5z6mvcmga3gx2toe6bv2gtd5n1scqc5ic&amp;ep=v1_gifs_search&amp;rid=giphy.gif&amp;ct=g" style="border:0;height:300px;width:300px"/>